modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -719,6 +738,65 @@ private void _addObjectDefinitionResources(
 		}
 	}
 
+	private Set<String> _getAccountEntryRestrictiveObjectRelationshipsNames(
+		ObjectRelationship[] objectRelationships,
+		com.liferay.object.model.ObjectDefinition
+			serviceBuilderObjectDefinition) {
+
+		if ((objectRelationships == null) ||
+			(serviceBuilderObjectDefinition == null) ||
+			!StringUtil.equals(
+				serviceBuilderObjectDefinition.getName(), "AccountEntry")) {
+
+			return Collections.emptySet();
+		}
+
+		Set<String> accountEntryRestrictiveObjectRelationshipsNames =
+			new HashSet<>();
```

Comment:
@devpedrotavares please rename to `accountEntryRestrictedObjectRelationshipNames` (`Restricted` instead of `Restrictive`)